src/comm/aca_comm_mgr.cpp

```diff
@@ -102,6 +102,14 @@ int Aca_Comm_Manager::update_goal_state(GoalState &parsed_struct,
     rc = exec_command_rc;
   }
 
+  exec_command_rc = Aca_Dhcp_State_Handler::get_instance().update_dhcp_states(
+          parsed_struct, gsOperationReply);
```

Comment:
looks like you have the latest changes since parsed_struct and been renamed to goal_state_message. This may have cased the build break on this PR. Please update your master with the latest.